com/artbeatte/exercises/bowlingball/BowlingBallTester.java

```diff
@@ -0,0 +1,85 @@
+package com.artbeatte.exercises.bowlingball;
+
+import com.artbeatte.exercises.testing.ExternalTestCase;
+import com.artbeatte.exercises.testing.SystemTestRunner;
+import com.artbeatte.exercises.testing.TestRunner;
+
+import java.io.IOException;
+import java.io.OutputStream;
+
+/**
+ * Created by sarbs on 11/16/15.
+ */
+public class BowlingBallTester {
+
+    public static void main(String args[]) {
```

Comment:
Because these are Exercises was there a reason why the tester, being just a wrapper for `public static void main(...)`, is not just part of the Exercise class? (like the other exercises). Its not really the style of everything else. However, if you can make a good enough case Id be willing to change.